app/full_text_search.py

```diff
@@ -113,52 +116,93 @@ def search(repo, query_txt, max_results):
     return results
 
 
-def create_jp_name_fields(**kwargs):
+def create_full_name_without_space(name_component1, name_component2):
```

Comment:
If you return a tuple, then naming given_name, family_name is better because you always pass these two in this order :)